src/resolvers/projectResolver.test.ts

```diff
@@ -1065,6 +1049,106 @@ function editProjectTestCases() {
       createProjectResult.data.data.addProject.walletAddress,
     );
   });
+  it('Should update successfully and verified(true) field would not change', async () => {
+    const accessToken = await generateTestAccessToken(SEED_DATA.FIRST_USER.id);
+    const project = await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      verified: true,
+    });
+    const editProjectResult = await axios.post(
+      graphqlUrl,
+      {
+        query: editProjectQuery,
+        variables: {
+          projectId: project.id,
+          newProjectData: {
+            title: String(new Date().getTime()),
+          },
+        },
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      },
+    );
+    assert.isTrue(editProjectResult.data.data.editProject.verified);
+  });
+  it('Should update successfully and verified(true) field would not change', async () => {
```

Comment:
Just a typo, the test indicated verified(true) in the description but its actually false if you look at the test.

Reply:
Oh you are right, It has been fixed
Thank you for being so punctilious